docs/flank-github-action/flank_github_acton_sdd.md

```diff
@@ -49,25 +49,25 @@ Flank Github action will be developed using [composite run steps action](https:/
 name: 'Flank'
 description: 'Run Flank from Github actions!'
 inputs:
-	version: 
-		description: 'Version of flank to run'
-		required: true
-		default: <latest>
-	platform: 
-		description: 'Platform to run iOS or Android'
-		required: true
+    version: 
+        description: 'Version of flank to run'
+        required: true
+        default: <latest>
+    platform: 
+        description: 'Platform to run iOS or Android'
+        required: true
   flank_option1:
     description: 'Description of option 1'
     required: false # validation will done when running
     ...
-  flank_optionX:
-    description: 'Description of option X'
+  flank_optionN:
+    description: 'Description of option N'
     required: false # validation will done when running
   flank_configuration_file:
     description: 'Path to configuration file'
     required: false # validation will done when running
-	flank_service_account: 
```

Comment:
How will authorization work? User is responsible for creating authorization json in the separate step and then falnk will use its output (?)

Reply:
Hey please check https://github.com/Flank/flank/issues/1597
The user could create a file or pass a file content itself